packages/design-system/src/components/contextMenu/menu.js

```diff
@@ -41,7 +41,7 @@ const MenuWrapper = styled.div(
     pointer-events: none;
     border-radius: ${theme.borders.radius.small};
     border: 1px solid ${theme.colors.border.disable};
-    width: ${({ isIconMenu }) => (isIconMenu ? '40px' : '200px')};
+    width: ${({ isIconMenu }) => (isIconMenu ? '40px' : '210px')};
```

Comment:
just curious - why the 10px?

Reply:
yea Just added that because the labels for the bottom two actions are a little long and this prevents them from wrapping

Reply:
i will say, it'll only help in english. 😹 we should be braced for some wrap potentially (which doesn't need to get handled here, just mentioning)